contrib/compose-rhcs.sh

```diff
@@ -6,7 +6,16 @@ set -e
 # VARIABLES #
 #############
 
-STAGING_DIR=staging/main-ubi9-latest-x86_64/
```

Comment:
To make this less error-prone, we could hard-code the RHCS version logic here, and present a single `VERSION` variable to users:

* `VERSION=5` would be mean `pacific` and `ubi8`
* `VERSION=6` would mean `quincy` and `ubi9`